w11h03/test/pgdp/networking/DataHandlerTest.java

```diff
@@ -0,0 +1,60 @@
+package pgdp.networking;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static pgdp.networking.Util.*;
+
+public class DataHandlerTest {
+
+    private static final String USERNAME = "TestUser123";
+    private static final String ID = "geXYZtup";
```

Comment:
```suggestion
    private static final String ID = "ge69sex";
```
it’s important to follow digits and and the ending of format consonant vowel consonant

Reply:
If you say so xD